index.html

```diff
@@ -254,7 +254,23 @@ <h1 id="block-apart-primary-text-1-project-title-text-4" class="block-apart-prim
 
 
 
+    <!-- contact form -->
+    <section id="contact-form-base"></section>
+    <section id="contact-form-shapes"></section>
+    <section id="contact-form-data">
+      <h1 id="contact-form-data-h1">Contact me</h1>
+      <h6 id="contact-form-data-h6">If you have an application you<br> are interested in developing, a<br> feature that you need built or a<br> project that needs coding. I’d<br> love to help with it </h6>
 
+      <div id="shape"></div>
+
+        <form id="survey-form" action="https://formspree.io/f/mpznrolg" method="post">
+          <input id="survey-form-name-input" name="Name" type="text" required="true" required placeholder="name" maxlength="30">
+          <input id="survey-form-email-input" name="email" type="email" required="true" required placeholder="email">
+          <textarea id="survey-form-message-input" name="Message" type="text" required="true" required placeholder="Write your message here" maxlength="500"></textarea>
+          <label id="message-alert"></label>
+          <button id="contact-form-data-button" type="submit">Get It Touch</button>
+        </form>
+    </section>
```

Comment:
- [ ] Good job team :+1: . You stick as much as possible to the Figma design. However, You should use flexblox to layout the elements of the form on the page. Kind make sure you follow [this requirement.](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=In%20order%20to%20lay%20out%20the%20elements%20on%20the%20page%20you%20should%20use%20Flexbox) :100: